Sources/LiveViewNative/Modifiers/Toolbars Modifiers/ToolbarVisibilityModifier.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ToolbarVisibilityModifier.swift
+//  LiveViewNative
+//
+//  Created by Carson Katri on 3/30/2023.
+//
+
+import SwiftUI
+
+/// Sets the visibility of a toolbar.
+/// 
+/// ```html
+/// <Text modifiers={
+///     toolbar(content: "my-toolbar")
+///     |> toolbar_visibility(visibility: :hidden, bars: :)
```

Comment:
The `native` assign seems to be missing from this documentation and it seems like the `bars` param is missing.

```suggestion
///     @native
///     |> toolbar(content: :"my-toolbar")
///     |> toolbar_visibility(visibility: :hidden, bars: :bottom_bar)
```